### ISSUE TITLE
Implement Wrong Letter Screen

### ISSUE THREAD

**Issue by thellmund** [serlo-org/serlo-abc#18]

**Comment by zepatrik** [comment]
This picture?
![22](https://cloud.githubusercontent.com/assets/5354445/25255195/c35fbab2-2629-11e7-85fa-9d822fd32587.jpg)
Or this?
![18](https://cloud.githubusercontent.com/assets/5354445/25255328/3b1daadc-262a-11e7-991e-510176289073.jpg)
Should it randomly pic a wrong letter? I think controlling where and what the wrong letter is can be used to set a difficulty.

**Comment by inyono** [comment]
See discussion in #91. All exercises should be implemented without randomness since that will be solved later by a higher-order component. So it's fine if you include a story with a fixed wrong letter passed as a prop for now.

**Comment by zepatrik** [comment]
Ok but still I'm not quite sure what this exercise should look like
Maybe @ronnitatserlo can make it clear?
Are the screenshots above the right ones?

**Comment by ronnitatserlo** [comment]
Yes, the screenshots above are correct, it is a spelling exercise. The idea being that words which have already been introduced in any lection will be taken in this exercise, read out loud and displayed with spelling mistakes. The user is supposed to recognise from the soundfile which letter in the word is wrong and replace it with the correct letter. I guess it is important in these exercises to only replace letters that have already been learned within the app up to that point, as some words will also have letters that have not yet been introduced to the learners. Would you agree, @andreas-serlo?

**Comment by inyono** [comment]
To extend @ronnitatserlo's comment: for now it's okay if you show the image, play the sound and show the wrongly spelled word. The user should then be able to select he wrong letter (resp. letters, see the different difficulty levels on https://de.serlo.org/abc/ex-detail and the prototype demo https://abc.serlo.org/try/#802) and replace it with the correct one(s). The letters that are selectable are also passed to the component and will later be handled by #91.
